Explorer/Assets/Scripts/CRDT/Protocol/Factory/ProcessedCRDTMessage.cs

```diff
@@ -16,5 +17,55 @@ public ProcessedCRDTMessage(CRDTMessage message, int crdtMessageDataLength)
             this.message = message;
             CRDTMessageDataLength = crdtMessageDataLength;
         }
+
+        public override string ToString() =>
+            $"Message: {message} CRDTMessageDataLength: {CRDTMessageDataLength}";
+
+        public void LogSelf(string fromPlace)
```

Comment:
I would put the whole method under a custom `Conditional` attribute, not `UNITY_EDITOR` as it will lead to significant performance disparity between a build and editor. Thus, you don't really need all the interfaces: only a single implementation in this method.